static/bidder-params/adpone.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Adpone Adapter Params",
+  "description": "A schema which validates params accepted by the adpone adapter",
+  "type": "object",
+  "properties": {},
```

Comment:
Are we sure we don't have any parameters we want to check for? If not, do you think you'll need them in the future?

Reply:
Probably in the future we will need some parameters so I have added one (placementId). Thanks!